editor/property_editor.cpp

```diff
@@ -882,7 +882,13 @@ bool CustomPropertyEditor::edit(Object *p_owner, const String &p_name, Variant::
 			} else if (hint_text != "") {
 				int idx = 0;
 
-				const Vector<EditorData::CustomType> custom_resources = EditorNode::get_editor_data().get_custom_types()["Resource"];
+				Vector<EditorData::CustomType> custom_resources;
+
+				if (EditorNode::get_editor_data().get_custom_types().has("Resource"))
+				{
```

Comment:
Brace should be on the same line as `if`

Reply:
@Noshyaar I fixed it, check now, its safe to merge